homeassistant/config.py

```diff
@@ -608,7 +608,7 @@ def merge_packages_config(hass: HomeAssistant, config: Dict, packages: Dict,
         for comp_name, comp_conf in pack_conf.items():
             if comp_name == CONF_CORE:
                 continue
-            component = get_component(hass, comp_name)
+            component = get_component(hass, comp_name.split(' ')[0])
```

Comment:
You could add a comment about the reason for the split or maybe extract the result to a new variable so the variable name could describe it.

Reply:
Good point. It's kind of hacky, so it could do with an explanation.